### PR TITLE
[Test] Run copy-propagation on SILOptimizer/opaque_values_mandatory.sil.

### PR DESCRIPTION
**Pull request by nate-chandler**

### CODE REVIEW

**Comment by nate-chandler** [comment]
@swift-ci please test

**Review by atrick — APPROVED** [review]
This run-line looks good to me.

 -enable-ossa-modules currently enables non-mandatory copy propagation at -O only just so we can get a good performance baseline.

File: tools/sil-opt/SILOpt.cpp [hunk 1]
```diff
@@ -122,6 +122,10 @@ static llvm::cl::opt<bool> EnableOSSAModules(
                    "this is disabled we do not serialize in OSSA "
                    "form when optimizing."));
 
+static llvm::cl::opt<bool> EnableCopyPropagation(
```
Comment (atrick) [inline tools/sil-opt/SILOpt.cpp:125]: We actually have two options -enable-copy-propagation and -disable-copy-propagation, both of which default to 'false'. Enabling either one overrides the default pipeline behavior, which depends on the optimization level and whether OSSA is enabled.

Comment (nate-chandler) [reply tools/sil-opt/SILOpt.cpp:125]: It looks like swift-frontend has those flags to control the `EnableCopyPropagation` and `DisableCopyPropagation` flags of `the `SILOptions` class sil-opt doesn't.  Considering that they both have independent effects, it seems likely that at some point they'll both be useful for `sil-opt`.  I went ahead and pushed another commit to add it.

The test failures were
```
  Swift(macosx-x86_64) :: Interpreter/SDK/objc_bridge_cast.swift
  Swift(macosx-x86_64) :: Interpreter/SDK/objc_dealloc.swift
```
and seem to be affecting unrelated, affecting other PRs, like https://ci.swift.org/job/swift-PR-macos/28081/consoleText .

**Comment by nate-chandler** [comment]
@swift-ci please test

**Review by atrick — APPROVED** [review]
LGTM

**Comment by nate-chandler** [comment]
@swift-ci please test windows platform

**Comment by nate-chandler** [comment]
@swift-ci please test windows platform

**Comment by nate-chandler** [comment]
@swift-ci please test windows platform

**Comment by nate-chandler** [comment]
@swift-ci please clean test windows platform

**Comment by nate-chandler** [comment]
@swift-ci please clean test windows platform

**Pull request merged**